components/AccountVerificationForm/AccountVerificationFormStep1PreConsent.js

```diff
@@ -168,14 +87,10 @@ export function AccountVerificationFormStep1PreConsent() {
         </p>
 
         {/* ACTIONS */}
-        <div className="space-y-2">
-          <Button variant="bold" block onClick={(() => goToConsent())}>
+        <div className="w-full mx-auto space-y-2 sm:w-64">
+          <Button variant="bold" block loading={submitting} disabled={submitting} onClick={(() => { goToConsent(); setSubmitting(true)})} data-cy="current-step">
```

Comment:
If you put this logic in another function, it would be nice
👇 
{ goToConsent(); setSubmitting(true) }